File: BC - w4/Basic/complex/main.cpp
#include <iostream>
#include "complex.h"
#include <cmath>

    int main() {
    Complex a(1.2, -3.4);
    Complex b(5.6, 7.8);



    std::cout << a << b<< std::endl;
    std::cout << a * b<< std::endl;


    delete(&a);
    delete(&b);

    return 0;

}

File: BC - w4/java/Basic/Complex/Main.java
package Complex;


public class Main {

	public static void main(String[] args) {
		Complex a = new Complex(5, 10);
		Complex b = new Complex (2, 5);
		b.print();
		
		b.increm(a);
	

		
		a.print();
		b.print();

	}

}
